src/site/content/en/blog/webassembly-threads/index.md

```diff
@@ -0,0 +1,472 @@
+---
+title: Using WebAssembly threads from C, C++ and Rust
```

Comment:
```suggestion
title: Using WebAssembly threads from C, C++ and Rust
subhead: Learn how to bring multithreaded applications written in other languages to WebAssembly.
```

Reply:
Should it really duplicate the description? Is one derived from the other?

Reply:
Description is for search engines, should be 1-3 sentence summary of the post. The subhead is what's displayed on the blog page, so it's a chance to give a bit more info than what fits in the title. They can be identical, though description could be more descriptive too. :)